deployment/chainloop/values.yaml

```diff
@@ -919,6 +928,15 @@ cas:
     grpc: 9000
     metrics: 5000
 
+  ##
+  ## TLS configuration
+  ##
```

Comment:
please update the description to clearly explain what this option is for, similar to what we had before

> name of a secret containing TLS certificate to be used by the controlplane grpc server. The secret must contains 2 keys: tls.crt and tls.key respectively containing the certificate and private key.